### PR TITLE
Feature/rebranding

### PR DESCRIPTION
**Pull request by dex4er**
All files rebranded. It uses freelens repo for building. The pipeline is enabled for linter and integration tests.

I haven't touched on "publishing" yet.

### CODE REVIEW

**Comment by robertobandini** [comment]
It seems that obviously we have first to publish the packages needed from the freelens repository, the core and the extension.
Just evaluating if re-enable the npm publish action on the freelens repo.
Maybe we also should re-enable the tests here on this repo.

**Review by robertobandini — APPROVED** [review]

**Comment by dex4er** [comment]
No, we do not need to publish core repositories. Freelens does not need them as they are already installed for the application's environment.

The pipeline git clones the repository with Freelens to build this extension so it works without npmjs.org.

I think it might be needed only if you want to install this extension by its name rather than tar.gz package.

**Review by robertobandini — APPROVED** [review]

**Comment by robertobandini** [comment]
Ok, let's verify, so if Freelens packages are needed for other extensions and we want to maintain the extension api possibility maybe we'll have to publish again.

File: integration/__tests__/extensions.tests.ts
Comment (robertobandini) [inline integration/__tests__/extensions.tests.ts:1]: Were they failing also before this issue?

Comment (dex4er) [reply integration/__tests__/extensions.tests.ts:1]: Yes, it failed even before rebranding.

**Pull request merged**